plugins/woocommerce/client/admin/client/marketplace/marketplace.scss

```
@@ -1,5 +1,11 @@
 @import "./stylesheets/_variables.scss";
 
+.woocommerce-admin-page__extensions #wpcontent,
+.woocommerce-admin-page__extensions.woocommerce_page_wc-admin #wpbody-content {
+	overflow-x: inherit !important; // Overwriting wc-admin css from elsewhere. Necessary to make the faetures banner 'stick
+	position: relative;
+}
+
 .woocommerce-admin-page__extensions {
 	background: #fff;
 
```

Any reason this code isn't wrapped in the scope that's next to it, like most of the rest of the code is?

If you think it's more-visible where it is, and that's important, then I guess that's fine... it just feels a bit odd!

```suggestion
.woocommerce-admin-page__extensions {
	background: #fff;

	#wpcontent,
	&.woocommerce_page_wc-admin #wpbody-content {
		overflow-x: inherit !important; // Overwriting wc-admin css from elsewhere. Necessary to make the faetures banner 'stick
		position: relative;
	}

```

(Not tested, just observing.)

---

Great suggestion Dan, fixed it in [0e65302](https://github.com/woocommerce/woocommerce/pull/55904/commits/0e65302ebfe0fc436cac55ecb73e41b9362ce926)